docs/reference/api-gateway/http.md

```diff
@@ -0,0 +1,171 @@
+# API Gateway with http requests
+
+#### In this document
+
+- [Without Authentication](#none-auth)
```

Comment:
Change to "No authentication" as it express better there is no authentication ;)